modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/discovery/impl/DiscoveryServiceImpl.java

```diff
@@ -109,4 +100,37 @@ public Response discoverGet() {
         }
         return Response.ok().build();
     }
+
+    /**
+     * Get authentication policy from the tenant configuration, otherwise set default value as Federated.
+     *
+     * @return Authentication policy.
+     * @throws WindowsDeviceEnrolmentException
+     */
+    public String getAuthPolicy() throws WindowsDeviceEnrolmentException {
```

Comment:
better having a method name like getAuthPolicyType rather than getAuthPolicy as this does not actually returns some kind of policy.